arturito/src/components/PeopleSection/index.tsx

```diff
@@ -0,0 +1,48 @@
+import useSWR from 'swr';
+
+import { swGet } from '../../utils/fetcher';
+import Table from '../Table';
+
+const columns = [
+    {
+        title: 'Name',
+        dataIndex: 'name',
+        key: 'name',
+    },
+    {
+        title: 'Birth_year',
+        dataIndex: 'birth_year',
+        key: 'birth_year',
+    },
+    {
+        title: 'Height',
+        dataIndex: 'height',
+        key: 'height',
+        render: (height: string) =>(parseInt( height) / 100) +' centimetros',
```

Comment:
Please, change "centimetros" to "centimeters".

Reply:
Checkuot again bro